janusgraph-core/src/main/java/org/janusgraph/graphdb/util/PriorityQueue.java

```diff
@@ -79,7 +79,7 @@ public PriorityQueue(int maxSize, boolean prepopulate) {
 
   /** Determines the ordering of objects in this priority queue.  Subclasses
    *  must define this one method.
-   *  @return <code>true</code> iff parameter <tt>a</tt> is less than parameter <tt>b</tt>.
+   *  @return <code>true</code> iff parameter <code>a</code> is less than parameter <code>b</code>.
```

Comment:
The change from `<tt>` to `<code>` seems to be unrelated to the stated goal of bumping the version of vavr, but the change itself is fine for consistency, so no need to factor this out, unless you didn't mean to commit this delta.

Reply:
I did execute JanusGraph yesterday with java 11 during my try to test these changes. It failed since tt is deprecated.